src/NServiceBus.Metrics.ServiceControl/IReportNativeQueueLength.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace NServiceBus.Metrics.ServiceControl
+{
+    using System.Collections.Generic;
+    using global::ServiceControl.Monitoring.Data;
+
+    /// <summary>
+    /// Reports native queue length to ServiceControl Monitoring
+    /// </summary>
+    public interface IReportNativeQueueLength
+    {
+        /// <summary>
+        /// Reports the length of a queue to be sent to ServiceControl Monitoring
+        /// </summary>
+        void ReportQueueLength(string physicalQueueName, long queueLength);
+
+        /// <summary>
+        /// A list of queues that are monitored
+        /// </summary>
+        IEnumerable<string> MonitoredQueues { get; }
```

Comment:
What is this collection for? It's exposed and never used. It looks like that was attached to the reporter interface.